data/mods/Globally_Unique_Specials/overmap/overmap_special/specials.json

```
@@ -0,0 +1,62 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "Isherwood Farms",
+    "copy-from": "Isherwood Farms",
+    "flags": [ "FARM", "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "evac_center",
+    "copy-from": "evac_center",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN", "MAN_MADE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "hub_01",
+    "copy-from": "hub_01",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "isolated_road",
+    "copy-from": "isolated_road",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "ranch_camp",
+    "copy-from": "ranch_camp",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN", "MAN_MADE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "valhalla_cult",
+    "copy-from": "valhalla_cult",
+    "flags": [ "CLASSIC", "GLOBALLY_UNIQUE", "MAN_MADE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Occupied Lumbermill",
+    "copy-from": "Occupied Lumbermill",
+    "flags": [ "CLASSIC", "GLOBALLY UNIQUE", "WILDERNESS" ]
```

```suggestion
    "flags": [ "CLASSIC", "UNIQUE", "WILDERNESS" ],
    "occurrences": [ 15, 100 ]
```
The occurrence chances we discussed earlier.